core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/runtimeRoutesModels.scala

```diff
@@ -149,6 +149,8 @@ object GetRuntimeResponse {
   )
 }
 
+final case class ListRuntimeIdResponse(id: Long, samResource: RuntimeSamResourceId)
```

Comment:
Still going through the PR.. but this file is really for models of the Leo API request/response JSON. it seems `ListRuntimeIdResponse` is more for DB deserialization, maybe there is another place for this model to go.

(Will update back if with suggestions)

Reply:
hm.. I was cleaving to the pattern used by the ListRuntime2 object :/ lmk!

Reply:
I think the JSON encoder/decoder for `ListRuntimeIdResponse` is not actually needed (at least, the code compiles for me without it). So maybe you can remove the encoders, and put `ListRuntimeIdResponse` in `samModels.scala`.

Reply:
Done!